ipc/cli/src/commands/checkpoint/relayer.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright 2022-2023 Protocol Labs
+// SPDX-License-Identifier: MIT
+
+use crate::commands::get_subnet_config;
+use crate::{CommandLineHandler, GlobalArguments};
+use anyhow::anyhow;
+use async_trait::async_trait;
+use clap::Args;
+use fvm_shared::address::Address;
+use ipc_identity::EvmKeyStore;
+use ipc_provider::checkpoint::BottomUpCheckpointManager;
+use ipc_provider::new_evm_keystore_from_path;
+use ipc_sdk::subnet_id::SubnetID;
+use std::str::FromStr;
+use std::sync::{Arc, RwLock};
+use std::time::Duration;
+
+const DEFAULT_CHECKPOINT_INTERVAL: u64 = 15;
```

Comment:
This is not the checkpoint interval but the polling time  in the process loop, right? I think we should rename it and clarify in a comment.
```suggestion
const DEFAULT_POLLING_INTERVAL: u64 = 15;
```